src/pages/user/User.tsx

```diff
@@ -57,10 +57,10 @@ const User: React.FC = () => {
   }, [dispatch, location.hash, history]);
 
   useEffect(() => {
-    if (user.orcid && !user.authority?.orcId) {
-      updateAuthority(user.authority, dispatch);
+    if (user.authority?.orcids) {
```

Comment:
skal det egentlig være en sjekk på length her? for denne vil vel bli true om `orcids = []` også, men da vil `orcids[0]` etterpå gi error

Reply:
good catch 👍  fikser